### DISCUSSION TITLE
Refresh button

### DISCUSSION THREAD

**Discussion by ibrahimsel** [foxglove/community#837] [General]
Does the app have a refresh (or a restart) button that I'm not aware of? If not, it'd be so nice to have one. Everytime something in the visual display bugs, it's so time consuming having to restart the app manually all over

**Comment by jtbandes** [comment]
If you experience visual bugs, please report them at https://github.com/foxglove/studio/issues/new/choose — we strive to fix or at least mitigate any bugs that arise.

**Comment by defunctzombie** [comment]
Closing this as asnwered.
